frontend/packages/component-library/package.json

```diff
@@ -355,7 +355,6 @@
     "@code-dot-org/changelogs": "workspace:*",
     "@code-dot-org/component-library-styles": "workspace:*",
     "@code-dot-org/lint-config": "workspace:*",
-    "@contentful/experiences-sdk-react": "1.30.3",
```

Comment:
This was an unused dependency (as we had moved definitions to the marketing app)